src/com/hms_networks/americas/sc/mqtt/package.html

```diff
@@ -2,7 +2,7 @@
 <BODY>
 Ewon Flexy MQTT library created by the HMS Networks MU Americas Solution Center.
 
-@version 1.1
+@version 1.2
```

Comment:
This should be a separate commit titled something like "Rev'd version number to v1.2"